.github/workflows/consistency-test.yml

```diff
@@ -0,0 +1,68 @@
+name: consistency_testing
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: ref to fetch package for
+        required: true
+jobs:
+  run_consistency_tests:
+    runs-on: ubuntu-20.04
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      CI: 1
+      CI_INSTALL_INFRA: 1
+      AWS_EC2_METADATA_DISABLED: true
+
+    steps:
+      - name: gcloud auth
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          project_id: 'redpandaci'
+          service_account_key: ${{ secrets.GCR_JSON}}
+          export_default_credentials: true
+
+      - name: fetch release from gcb
+        run: |
+          gsutil cp gs://vectorizedio/rp/release/clang/${{ github.event.inputs.ref }}/redpanda_*.deb redpanda.deb
```

Comment:
what is inputs.ref? this is testing when we tag? or when there is a command? 

what triggers this. could it be that a command would do a full release pipeline and then test that request itself? 

ok to defer that feature to a seaparate project. 

also let's make sure that the aws buckets are authenticated too.

Reply:
The bucker is configurable so we can create it with any permissions we like (as long as the AWS creds we provide have write access).

Currently the workflow runs manually and needs the commit hash of a build from GCB. It definitely is UX as bad as it gets.
I'm creating a postback to github from GCB, so we can trigger github actions when custom events happen (eg `package_created`, `release_published` etc). This will probably prove useful for other actions too (eg integration tests, test deployments).